static/Gruntfile.js

```diff
@@ -1,5 +1,5 @@
 const webpackConfig = require('./webpack-config');
-const { exec } = require("child_process");
+const { spawnSync } = require("child_process");
```

Comment:
single quotes here! (I think there's some way to make the vscode autoimport use single quotes)